### ISSUE TITLE
Windows : Badge Notification on Icon tray

### ISSUE THREAD

**Issue by audelorAd** [nextcloud/desktop#4348]
### Feature description
When we get new notification on nextcloud its easy to missed it with Desktop Client.
Desktop client use windows notification system, so we can see notification if we are behind our computer.
If we are not its hard to see that we have a new notification.

A badge notification above Nextcloud icon tray could visually warn on a new notification.

Something like that
![badge_nextcloud](https://user-images.githubusercontent.com/63049494/158209481-82a2e4c3-94a0-4d69-b000-05f0f56c671d.png)

Or like that
![badge_nexcloud_2](https://user-images.githubusercontent.com/63049494/158209476-59686cae-4b54-4da0-8f21-f4f9902daaea.png)

**Comment by splitt3r** [comment]
I think Windows already shows this information on the notification center icon (the most right one). Doesn't it?

**Comment by audelorAd** [comment]
Yes indeed we can found this information on the Windows notification center.
But a lot of software use this notification center so that we can quickly have tens of notification here.
I open this issue because for me software notification badge is more handy for a today use than the global Windows notification center.

**Comment by HadesARCH** [comment]
Hi,
I'm totaly agree with you @audelorAd, i don't look at the windows notifications because praticly all notifications are useless or producted by many software. And it's more confortable when you are not sollicited by notificatons for focus on work.
I use Thunderbird who have this red badge and it's very usefull because you see if you have new mail even if you don't have notification. 
I hope this fonctionnality will be implented soon on nextcloud.
@splitt3r do you know if someone can do this ? We don't have knowledge in developpement and we can paid for this funcionnality for our company.
Thank's

**Comment by poiNt3D** [comment]
Yes, it would be very handy, especially if it would show notifications for unread messages in Talk, and remove the icon when messages are read.

**Comment by RTechSn** [comment]
This is a very frequently asked feature from users of my server.

**Comment by audelorAd** [comment]
According to me Windows Notification Center is not usuable, too many things in it.
Nextcloud Desktop notification should be way more visible than it is today.

Maybe by keeping Nextcloud Icon always visible in the bottom taskbar (see image bellow), and make icon flashing on a new unread notification.
![image](https://github.com/nextcloud/desktop/assets/63049494/d2acd3d4-9d6d-492b-ab44-830f457b967f)

**Comment by Rello** [comment]
The icons were re-done last year for better visibility.
